src/Phinx/Console/Command/Create.php

```diff
@@ -112,6 +114,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        $output->writeln('<info>using migration base class</info> ' . $this->getConfig()->getMigrationBaseClassName(false));
```

Comment:
Rather than doing a whole new `getMigrationBaseClassName` call here, why not use `$classes['$useClassName']`?